### DISCUSSION TITLE
Sıradaki şehir hangisidir?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#192] [Zeka Sorusu]
1- Bursa
2- Adana
3- Adıyaman
4- Nevşehir
5- ?
Sıradaki il hangisi?

**Comment by akgembi12** [comment]
Rize. Son harfleri ile basliyorlar

**Comment by TheDemr0** [comment]
Her şehir bir önceki şehirin son harfiyle başlıyor. O yüzden R ile başlayan bir şehir seçmeliyiz ve o şehirde Rize

**Comment by Onuralp000** [comment]
Son harfler ile başladığı için Rize oluyor.

**Comment by Sevvalgocgun1** [comment]
Her bir sehir bir onceki secenektekinin son harfiyle basliyor bu yuzden cevap:Rize
